### PR TITLE
fix(docs): left sidebar expand by default

### PR DESCRIPTION
**Pull request by ydcjeff**
Fixes https://github.com/pytorch/ignite/pull/1691#issuecomment-787473522

Description: `pytorch_sphinx_theme` has recently changes for navbar expand or collapse by default.
- Added `var collapsedSections = ['Notes']` in script tag to make `Notes` collapse by default
- Made padding = 0 to remove the gray footer
- Commented out three dot in mobile view as they don't do anything
- Commented out footer and `div class='container'` explicitly to have left side bar 100% height (you can still see the left sidebar when scrolling the engine page, on master docs, left sidebar disappear when scrolling to the end of engine page)

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [x] Documentation is updated (if required)

**NOTE : This PR is blocked by https://github.com/pytorch/pytorch_sphinx_theme/pull/97**

Preview output built locally
Screen Shot 2021-03-02 at 00 39 40

### CODE REVIEW

File: docs/source/_templates/layout.html [hunk 1]
```diff
@@ -121,13 +121,13 @@
           </li>
 
           <li>
-            <a href="{{ theme_variables.external_urls['home'] }}/master/about.html">About us</a>
+            <a href="{{ pathto('about') }}">About us</a>
```
Comment (vfdev-5) [inline docs/source/_templates/layout.html:124]: I think we should keep previous code as intention is to always show the recent info and not that one coined to the release...

File: docs/source/_templates/_static/css/ignite_theme.css [hunk 2]
```diff
@@ -168,3 +168,7 @@ footer table {
   width: 100%;
   margin-bottom: 1rem;
 }
+
+.pytorch-body .docs-tutorials-resources {
+  padding: 0 !important;
```
Comment (vfdev-5) [inline docs/source/_templates/_static/css/ignite_theme.css:173]: Can you please explain which gray footer this removes ?

File: ignite/contrib/handlers/lr_finder.py [hunk 3]
```diff
@@ -270,7 +270,7 @@ def attach(
 
             to_save = {"model": model, "optimizer": optimizer}
             with lr_finder.attach(trainer, to_save=to_save) as trainer_with_lr_finder:
-                trainer_with_lr_finder.run(dataloader)`
+                trainer_with_lr_finder.run(dataloader)
```
Comment (vfdev-5) [inline ignite/contrib/handlers/lr_finder.py:273]: Nice catch!

File: ignite/utils.py [hunk 4]
```diff
@@ -69,7 +69,7 @@ def apply_to_type(
 
 def to_onehot(indices: torch.Tensor, num_classes: int) -> torch.Tensor:
     """Convert a tensor of indices of any shape `(N, ...)` to a
-    tensor of one-hot indicators of shape `(N, num_classes, ...) and of type uint8. Output's device is equal to the
+    tensor of one-hot indicators of shape `(N, num_classes, ...)` and of type uint8. Output's device is equal to the
```
Comment (vfdev-5) [inline ignite/utils.py:72]: 👍

**Review by vfdev-5 — APPROVED** [review]
Thanks for the PR @ydcjeff !
Few questions and comments

File: docs/source/_templates/_static/css/ignite_theme.css (hunk 2 above)
Comment (ydcjeff) [reply docs/source/_templates/_static/css/ignite_theme.css:173]: It removes this gray footer
Screen Shot 2021-03-02 at 12 57 42

Comment (vfdev-5) [reply docs/source/_templates/_static/css/ignite_theme.css:173]: Thanks for the screenshot. Just to make sure, we do not see it currently neither on https://pytorch.org/ignite/ right ?

Comment (ydcjeff) [reply docs/source/_templates/_static/css/ignite_theme.css:173]: Yes, but if you see the left side bar when you scroll to the end of the engine page, the left side bar disappears with scrolling. To have the left side bar to stick, we need to enable `<div class="container-fluid docs-tutorials-resources" id="docs-tutorials-resources">` this line. When this line is enabled, that gray footer appears, so we need padding 0 around them.

Comment (vfdev-5) [reply docs/source/_templates/_static/css/ignite_theme.css:173]: Sounds good! Thanks for the explanation!

File: docs/source/_templates/_static/css/ignite_theme.css [hunk 5]
```diff
@@ -169,6 +169,11 @@ footer table {
   margin-bottom: 1rem;
 }
 
+
```
Comment (ydcjeff) [inline docs/source/_templates/_static/css/ignite_theme.css:172]:
```suggestion
```

File: docs/source/_templates/_static/css/ignite_theme.css [hunk 6]
```diff
@@ -197,4 +202,5 @@ footer table {
 .algolia-autocomplete .algolia-docsearch-suggestion--highlight {
   color: var(--red) !important;
   background-color: #ff0 !important;
+
```
Comment (ydcjeff) [inline docs/source/_templates/_static/css/ignite_theme.css:205]:
```suggestion
```

**Comment by vfdev-5** [comment]
@ydcjeff looks very neat now ! Thanks a lot for the improvements !

**Review by vfdev-5 — APPROVED** [review]

**Pull request merged**